Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -450,7 +450,7 @@
             "name": "isBelongCohort",
             "type": 1,
             "description": "Extract IsBelong property in cohort json from UsersList dropdown ",
-            "query": "let query = '{UsersList}';\r\n print parse_json(query).isBelong",
+            "query": "let query = '{UsersList:escape}';\r\nprint parse_json(query).isBelong",
```

Comment:
same on all of these.  if you're just doing this to parse json, there are better ways that don't require running a query at all

Reply:
same here. will update it.